schema/system_schema.md

```diff
@@ -0,0 +1,32 @@
+Define the default receiver to handle the case where no receiver is defined for a message. This ECU doesn't need JSON definitions and can be used to default to debug/logging purposes.
```

Comment:
could rename this to bus.json, that seems to be what its representing. system is pretty generic.

Reply:
good call

- [ ] rename to bus.json